contrib/remote_hsmd/NOTES.md

```diff
@@ -21,22 +11,49 @@ On Fedora:
 
     sudo dnf install -y grpc-devel grpc-plugins
 
-Building
+Python packages:
+
+    pip3 install --user base58 bitstring secp256k1 mrkd
+
+    # In c-lightning root, remote-hsmd branch:
+    pip3 install --user -r requirements.txt
+
+    # Temporarily downgrade markupsafe to avoid breaking dependencies
+    pip3 install --user markupsafe==2.0.1
+
 
+Validating Lightning Signer:
+
+    # In parent directory of c-lightning root
+    git clone git@gitlab.com:lightning-signer/validating-lightning-signer.git
+    cargo build --features log_pretty_print,debug_enforcement_state
+
+C-Lightning:
+
+    # Add an "upstream" reference and fetch all tags (needed for pyln-{proto,client,testing})
+    # Use the HTTPS pointer if you have authentication issues with git
+    git remote add upstream git@github.com:ElementsProject/lightning.git
+    git fetch --all --recurse-submodules
+    git fetch --tags
+
+    (cd contrib/remote_hsmd && \
+    ln -s ../../../validating-lightning-signer/lightning-signer-server/src/server/remotesigner.proto)
+    # Then do ls -alt contrib/remote_hsmd/remotesigner.proto and make sure the link is valid
+
+    # Build c-lightning
     make distclean
     ./configure --enable-developer
     make
 
-Build libsecp256k1 with `./configure --enable-module-recovery`, see
-https://github.com/golemfactory/golem/issues/2168 for background.
+On Ubuntu:
+
+    # Edit /etc/hosts, and add the entry localhost to the ::1 IP address, as below:
+    ::1     ip6-localhost ip6-loopback localhost
```

Comment:
This won't be necessary after we merge https://gitlab.com/lightning-signer/validating-lightning-signer/-/merge_requests/211, but it doesn't hurt either ...

Reply:
In that case, I'd prefer to remove these instructions once that PR is merged.